app/models/channel.rb

```diff
@@ -376,6 +381,10 @@ def register_channel_for_promo
     Promo::RegisterChannelForPromoJob.new.perform(channel: self)
   end
 
+  def create_channel_card
+    CreateUpholdChannelCardJob.perform_now(uphold_connection_id: publisher.uphold_connection&.id, channel_id: id)
```

Comment:
If Uphold goes down, anyone adding a card would experience a major slowdown too on any page renders. Preferable that we make this ` perform_later`.

Reply:
I was debating about this one, but i'm good with moving it to `perform_later`

https://github.com/brave-intl/publishers/pull/2094/commits/e95259299e1a0f477d13b779ab7d0072878f5bd0